### PR TITLE
Implement race and ethnicity mapping

### PR DESCRIPTION
**Pull request by analoo**
#### 🔗 Jira ticket
[Link](https://codeforamerica.atlassian.net/browse/CCAP-125)

#### ✅ Completion tasks

- [ ] Added relevant tests
- [ ] Meets acceptance criteria

### CODE REVIEW

File: src/main/java/org/ilgcc/app/pdf/NeedChildcareForChildren.java [hunk 1]
```diff
@@ -27,33 +29,40 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
     int iteration = 1;
     String earliestCCAPStart= "";
     for (var child : SubmissionUtilities.getChildrenNeedingAssistance(submission)) {
-      results.put(getUniqueKey(), new SingleField("childFirstName", (String) child.getOrDefault("childFirstName", ""), iteration));
-      results.put(getUniqueKey(), new SingleField("childLastName", (String) child.getOrDefault("childLastName", ""), iteration));
```
Comment (analoo) [inline src/main/java/org/ilgcc/app/pdf/NeedChildcareForChildren.java:31]: we need the keys in the map to be unique but using this unique key generator makes these fields more difficult to test so this is a cleanup.

File: src/main/java/org/ilgcc/app/utils/RaceEthnicityOption.java [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
-  NATIVE_HAWAIIAN("general.inputs.race-ethnicity.native-hawaiian-or-pacific-islander"),
-  WHITE("general.inputs.race-ethnicity.white"),
-  OTHER("general.inputs.race-ethnicity.other");
+  ASIAN("general.inputs.race-ethnicity.asian", "4"),
+  BLACK("general.inputs.race-ethnicity.black-or-african-american", "2"),
+  HISPANIC("general.inputs.race-ethnicity.hispanic-latino-or-spanish", "3"),
+  MIDDLE_EASTERN("general.inputs.race-ethnicity.middle-eastern-or-north-african", "O"),
+  NATIVE_AMERICAN("general.inputs.race-ethnicity.native-american-or-alaska-native", "5"),
+  WHITE("general.inputs.race-ethnicity.white", "1"),
+  OTHER("general.inputs.race-ethnicity.other", "O");
```
Comment (cram-cfa) [inline src/main/java/org/ilgcc/app/utils/RaceEthnicityOption.java:11]: It's so weird to me that their mapping for ethnicity, and nothing else, is this random number system.

**Review by cram-cfa — APPROVED** [review]

**Comment by cram-cfa** [comment]
Still looks good to me, and I like the cleanup work you did!

File: src/main/java/org/ilgcc/app/utils/RaceEthnicityOption.java (hunk 2 above)
Comment (analoo) [reply src/main/java/org/ilgcc/app/utils/RaceEthnicityOption.java:11]: I absolutely agree @cram-cfa

**Pull request merged**